vaadinfrontend/src/main/java/life/qbic/datamanager/views/project/view/components/ProjectDetailsComponent.java

```diff
@@ -92,16 +120,40 @@ private final class Handler {
 
     private final ProjectDetailsComponent component;
     private final ProjectInformationService projectInformationService;
+    private final PersonSearchService personSearchService;
 
     private ProjectId selectedProject;
 
     public Handler(ProjectDetailsComponent component,
-        ProjectInformationService projectInformationService) {
+                   ProjectInformationService projectInformationService, PersonSearchService personSearchService) {
       this.component = component;
+
       this.projectInformationService = projectInformationService;
+      this.personSearchService = personSearchService;
+
       setFieldsEditableOnlyOnFocus();
       attachSubmissionActionOnBlur();
       restrictInputLength();
+      setUpPersonSearch(component.projectManagerComboBox);
+      contactClickListener();
+    }
+
+    private void contactClickListener() {
+      //when clicking the layout switch!
+      component.projectManagerLayout.addClickListener(click ->{
+        component.pmElement.setVisible(false);
+        component.projectManagerComboBox.setVisible(true);
+      });
+
+      //when selecting a pm switch back to the label element
+      component.projectManagerComboBox.addValueChangeListener(click -> {
+        PersonReference reference = click.getValue();
+        component.pmElement.setContent(reference.fullName(),reference.getEmailAddress());
+
+        component.projectManagerComboBox.setVisible(false);
+        component.pmElement.setVisible(true);
+      });
```

Comment:
maybe it would be worth thinking about a focus and a blur listener instead. There is code in place to decide what to do on a blur event. We could make use of that.